unique_mcp/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.1] - 2025-12-17
+- Add server settings
+
+## [0.1.0] - 2025-12-5
```

Comment:
[Commit](https://github.com/Unique-AG/ai/commit/15b152d279682b2dc57cc61cea49a64f7ab9d0e1) that introduced this version